pkg/storage/sqlstorage/store_bench_test.go

```diff
@@ -90,10 +90,10 @@ func BenchmarkStore(b *testing.B) {
 					return
```

Comment:
I think it's time to remove all of those benchmarks, they don't make sense anymore and are not used.
What do you think?

Reply:
Yes, i'm agree.